### PR TITLE
docs: added another resolver example for DC and namespace failover

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
This PR adds an example for a datacenter failover while using namespaces

### CODE REVIEW

File: website/content/docs/connect/config-entries/service-resolver.mdx [hunk 1]
```diff
@@ -168,6 +168,56 @@ spec:
...
+
+```hcl
+Kind           = "service-resolver"
+Name           = "product-api"
+Namespace      = "primary"
+ConnectTimeout = "0s"
+Failover = {
+  "*" = {
+    Datacenters = ["dc2"]
+    Namespace = "secodary"
```
Comment (blake) [inline website/content/docs/connect/config-entries/service-resolver.mdx:185]:
```suggestion
    Namespace = "secondary"
```

File: website/content/docs/connect/config-entries/service-resolver.mdx [hunk 2]
```diff
@@ -168,6 +168,56 @@ spec:
 
 </CodeTabs>
 
+<EnterpriseAlert product="consul">
+  Failover to another datacenter and namespace
```
Comment (blake) [inline website/content/docs/connect/config-entries/service-resolver.mdx:172]:
```suggestion
  Failover to another datacenter and namespace for all service subsets.
```

**Review by blake — COMMENTED** [review]
I also think that it'd be helpful to have an example that explicitly shows failover working within a DC so that users are aware that federation is not required to use service & namespace failover. For example:

```hcl
Kind           = "service-resolver"
Name          = "product-api"
Namespace      = "primary"
ConnectTimeout = "0s"
Failover = {
  "*" = {
    Service = "product-api-backup"
    Namespace = "secondary"
  }
}
```

**Review by blake — APPROVED** [review]
LGTM. 👍

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/548326.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 79170d97318bdb750ea565e39eb3ce74c2b1a4b8 onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 79170d97318bdb750ea565e39eb3ce74c2b1a4b8 onto `release/1.11.x` succeeded!

**Pull request merged**